js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/AssetChecks.tsx

```diff
@@ -28,29 +29,41 @@ export const AssetChecks = ({
   lastMaterializationTimestamp: string | undefined;
 }) => {
   const queryResult = useQuery<AssetChecksQuery, AssetChecksQueryVariables>(ASSET_CHECKS_QUERY, {
-    variables: {
-      assetKey,
-    },
+    variables: {assetKey},
   });
   const {data} = queryResult;
   useQueryRefreshAtInterval(queryResult, FIFTEEN_SECONDS);
 
   const [openCheck, setOpenCheck] = useQueryPersistedState<string | undefined>({
     queryKey: 'checkDetail',
   });
+
   function content() {
     if (!data) {
       return <LoadingSpinner purpose="page" />;
     }
+    const assetNode = data.assetNodeOrError;
     const result = data.assetChecksOrError!;
+    if (assetNode?.__typename !== 'AssetNode') {
+      return <span />;
+    }
```

Comment:
Maybe this should go after the migration required check on the next line

Reply:
Ahh good call I like that